### ISSUE TITLE
[FE] Build Transformers for R&S paragraphs and nodes. 

### ISSUE THREAD

**Issue by brianalloyd** [department-of-veterans-affairs/va.gov-team#14797]
## Issue Description
_We need to build transformers for any(all) paragraphs and nodes, included in task section below_

We'll use the below task mark down as a checklist to validate transformer build for all content types/nodes have been accomplished.

The below is derived from CMS ticket [#15861](https://github.com/department-of-veterans-affairs/va.gov-team/issues/15861#issuecomment-722590115)

## Front end tasks
- [ ] `node-basic_landing_page`
- [ ] `node-checklist`
- [ ] `node-faq_multiple_q_a`
- [ ] `node-media_list_images`
- [ ] `node-media_list_videos`
- [ ] `node-q_a`
- [ ] `node-step_by_step`
- [ ] `node-support_resources_detail_page`
- [ ] `paragraph-alert_single`,
- [ ] `paragraph-audience_topics`,
- [ ] `paragraph-button`,
- [ ] `paragraph-contact_information`,
- [ ] `paragraph-lists_of_links`
- [ ] `paragraph-q_a_group`,
- [ ] `paragraph-rich_text_char_limit_1000`,
- [ ] `paragraph-step_by_step`,
- [ ] `paragraph-step`
- [ ] `taxonomy_term-audience_beneficiaries`,
- [ ] `taxonomy_term-lc_categories`,
- [ ] `taxonomy_term-products`
- [ ] `taxonomy_term-topics`

## Acceptance Criteria
- [ ] Transformers support CMS related templates, paragraphs and nodes.

**Comment by brianalloyd** [comment]
Closing this item and will build separate ticket for each item.

**Comment by kelsonic** [comment]
We'll need to add more content types too i think

**Comment by brianalloyd** [comment]
Thanks @kelsonic going to annotate here that we'll want to break this out into an epic and include a ticket for each content type in support of the FE transformers work.  We'll do this post planning discussion on 11/18/20.

**Comment by jenniferlee-dsva** [comment]
Do we need a separate ticket/epic also for building transformers for benefit hub content types as well? For example: hub landing pages, detail pages; and there are content components in the benefit hubs like number call-outs, div.feature blue boxes, and so on. I honestly don't know if these are unique content types that need their own transformer build, but just flagging it in case they do. 

FYSA - transformer ticket for[ forms product here: #16352 ](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/16352) for Zach and Marci's team.

**Comment by ncksllvn** [comment]
I _think_ all of the benefit hub content models already have transformers built because they predated the transformers concept

**Comment by jenniferlee-dsva** [comment]
>I think all of the benefit hub content models already have transformers built because they predated the transformers concept

That would be nice, but I'm not sure - or Platform may have built it a while back and it might have drift? 

 @brianalloyd - Could you verify with Megan and Dror, so we have a complete list of whatever these transformer thingeys are called for benefit hub content types?

**Comment by brianalloyd** [comment]
Yes, I'll verify and comment and update ticket accordingly.

**Comment by brianalloyd** [comment]
This is the response from the convo with Megan and Dror: 

> I believe the benefit hub transformers are in place, but that doesn't mean there won't be remaining issues that need to be addressed — I'm simply not sure. Once your team is up to speed on transformers we'll head into next steps:
Having your developers determine which transformers your team should own
Running the diff & fixing all discrepancies (between GraphQL and CMS Export Build) in those areas
I need to write up tickets for that — will try to get to it today!

I'm linking her into our Epic so they understand our breakdown and can help us determine if some of the defined content types can be removed based on previous implementation efforts and ownership 'rights' as defined above.

**Comment by meganhkelley** [comment]
👋 here's the list of transformers we made (just scaffolding to unblock ourselves — they are not complete) after y'all launched R&S, in case it's helpful https://github.com/department-of-veterans-affairs/va.gov-team/issues/15861#issuecomment-722590115

**Comment by brianalloyd** [comment]
Closing this as we have tickets representing transformer build.
